database/createOne.js

```diff
@@ -0,0 +1,29 @@
+const faker = require('faker');
+const generator = require('../helpers/generator.js');
+
+const item = () => {
```

Comment:
We could ask, but it feels like this should receive args from the endpoints to let you pass in your own creation info. i.e. if I `POST` I want to decide what I'm creating, not just that I'm creating a thing

Reply:
Your reasoning makes sense. I was thinking that the critical part of the POST was the request being sent from the client to the server rather who determines the actual content of the POST (user versus the code).